client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -505,6 +505,16 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         MOBILE_APP_LOCALE("mobileAppLocale", "en_US", false),
 
         /**
+         * This parameter represents the main activity of Android mobile appplication. By default it is always empty.
+         */
+        MOBILE_APP_MAIN_ACTIVITY("mobileAppMainActivity","", false),
```

Comment:
Since these new properties are Android only can we add "Android" to the property names here?